src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -162,7 +168,7 @@ public RobotMap() {
 		// Human inputs
 		Component.operatorStick = new CustomJoystick(Port.HumanInput.joystick);
 		Component.operatorStick.setDeadzone(HumanInterfaceConfig.JOYSTICK_DEADZONE);
-		Component.teensyStick = new CustomJoystick(Port.HumanInput.teensyStick);
+		Component.teensyStick = new TeensyController(Port.HumanInput.teensyStick, 30);
```

Comment:
Are there actually 30 buttons? Also, can you move the number to a constant, maybe HumanInterfaceConfig.